src/NuGet.Core/NuGet.Commands/VerifyCommand/VerifyArgs.cs

```diff
@@ -70,5 +70,16 @@ public string PackagePath
         /// If not empty, signer certificate fingerprint must match one in this list
         /// </summary>
         public IEnumerable<string> CertificateFingerprint { get; set; }
+
+        [Obsolete("This API does ignores NuGet.Config file or trusted-signers list")]
+        public VerifyArgs()
+        { }
+
+        public VerifyArgs(ISettings settings)
```

Comment:
I imagine that `Logger` can't be null when passed to `VerifyRunner`, so why is settings a constructor argument, rather than a property with a setter, like all the other args?